README.md

```diff
@@ -4,139 +4,75 @@
   <img height="48" src="https://static.openfoodfacts.org/images/logos/off-logo-horizontal-light.svg">
 </picture>
 
-# New Open Food Facts mobile app for Android and iPhone - Codename: "Smooth App"
+<br>
+
+# Smooth App : An app for Android and iPhone 
+
 [![SmoothApp Post-Submit Tests](https://github.com/openfoodfacts/smooth-app/actions/workflows/postsubmit.yml/badge.svg)](https://github.com/openfoodfacts/smooth-app/actions/workflows/postsubmit.yml)
 
-Latest commit deployed to App Stores: (Released on Sep 6 6:29 PM as Build 731 (3.13.1)) https://github.com/openfoodfacts/smooth-app/compare/v3.8.1...v3.13.1
+- Latest commit deployed to App Stores: [Released on Sep 6 6:29 PM as Build 731 - 3.13.1](https://github.com/openfoodfacts/smooth-app/compare/v3.8.1...v3.13.1)
```

Comment:
I know that that's not from you, but always updating it isn't really great

Could you link to the "latest" release

Reply:
Already applied this in my Commit